olive/evaluator/metric_config.py

```diff
@@ -53,6 +53,12 @@ def get_user_config_class(metric_type: str):
     return create_config_class(f"{metric_type.title()}UserConfig", default_config, ConfigBase, validators)
 
 
+def get_properties_from_metric_type(metric_type):
+    user_config_class = get_user_config_class(metric_type)
+    # list(user_config_class.schema()["properties"].keys()) # will ignore dataloader_func
```

Comment:
Nit: is this comment needed?

Reply:
Thanks. Actually I just do not know why the .schema() method ignored the dataloader_func even it was defined as schema. May some bugs or usage error of pydantic.

Reply:
How about to keep for reminding?

Reply:
`dataloader_func` is a Callable so it cannot be serialized to json schema. 
Ideally this comment has no relation to the existing codes but I'm okay to keep it if you think it is needed xd.

Reply:
> dataloader_func is a Callable so it cannot be serialized to json schema.
Oh, ic, so the pydantic schema will skip the field which cannot be serialized. I will remove it. Thanks. :)